e2e/nfs.go

```diff
@@ -371,80 +371,82 @@ var _ = Describe("nfs", func() {
 			})
 
 			By("verify RWOP volume support", func() {
-				err := createNFSStorageClass(f.ClientSet, f, false, nil)
-				if err != nil {
-					e2elog.Failf("failed to create CephFS storageclass: %v", err)
-				}
-				pvc, err := loadPVC(pvcRWOPPath)
-				if err != nil {
-					e2elog.Failf("failed to load PVC: %v", err)
-				}
-				pvc.Namespace = f.UniqueName
+				if k8sVersionGreaterEquals(f.ClientSet, 1, 22) {
```

Comment:
we're not testing with 1.21 anymore, this if-check can be removed so that the test is always enabled